### PR TITLE
[RPA 851] Hide actions while filtering

### PR DESCRIPTION
**Pull request by MRSIMM0**
## Description

When searching for actions guest was able to access action which should've been disabled.

image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx [hunk 1]
```diff
@@ -118,9 +117,12 @@ const ActionList: FC<ActionListProps> = ({
                                         item={item}
                                         disabled={
                                             !hasAdvancedActionsAccess &&
-                                            ADVANCED_ACTION_IDS.includes(
+                                            (ADVANCED_ACTION_IDS.includes(
                                                 item.id
-                                            )
+                                            ) ||
+                                                ADVANCED_ACTION_GROUP_IDS.includes(
```
Comment (MRSIMM0) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:123]: This is only significant change for this issue

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx [hunk 2]
```diff
@@ -118,9 +117,12 @@ const ActionList: FC<ActionListProps> = ({
...
+                                                    key
+                                                ))
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:125]: (ADVANCED_ACTION_IDS.includes(item.id) 
|| ADVANCED_ACTION_GROUP_IDS.includes(key))

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx [hunk 3]
```diff
@@ -2,13 +2,13 @@ import { VFC } from 'react';
...
-    url: string
+interface Props {
+    url: string;
 }
 
-const EmbeddedVideo: VFC<Props> = ({url}) => (
+const EmbeddedVideo: VFC<Props> = ({ url }) => (
     <div className={styles.video}>
-        <iframe loading='lazy' className={styles.embed} src={url} />
+        <iframe loading="lazy" className={styles.embed} src={url} />
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/EmbeddedVideo/EmbeddedVideo.tsx:11]: i think eslint allows only for singlequote but if it does not cause compilation errors it's fine

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx [hunk 4]
```diff
@@ -11,54 +14,106 @@ import EmbeddedVideo from '../EmbeddedVideo';
 import Typography from '../Typography';
 import styles from './RichTextRenderer.module.scss';
 
-
 interface Props {
     content: ContentfulRichText;
 }
 
-const rex = new RegExp('\\$https:\\/\\/www\\.youtube\\.com\\/watch\\?v=[^$]*\\$', 'g');
+const embeddedLinkRegex = new RegExp(
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:21]: dobra zmiana

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx [hunk 5]
```diff
@@ -11,54 +14,106 @@ import EmbeddedVideo from '../EmbeddedVideo';
...
             [BLOCKS.PARAGRAPH]: (_, children) => {
-
-                if( children.toString().match(rex)){
-                    const videoId  = children.toString().replace(/^\$/, '').replace(/\$$/, '').split('v=')[1];
+                if (children.toString().match(embeddedLinkRegex)) {
+                    const videoId = children
+                        .toString()
+                        .replace(/^\$/, '')
+                        .replace(/\$$/, '')
+                        .split('v=')[1];
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:68]: ❤️

File: runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx [hunk 6]
```diff
@@ -11,54 +14,106 @@ import EmbeddedVideo from '../EmbeddedVideo';
...
                     const url = `https://www.youtube.com/embed/${videoId}`;
-                    return (<EmbeddedVideo url={url}/>);
+                    return <EmbeddedVideo url={url} />;
```
Comment (star-in-the-jar) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:70]: :O

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx:70]: 👍

**Review by star-in-the-jar — APPROVED** [review]
przyznaj się, że prettiera włączyłeś

File: runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx (hunk 2 above)
Comment (MRSIMM0) [reply runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:125]: To akurat sprawka prettier XD

Comment (star-in-the-jar) [reply runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionList/ActionList.tsx:125]: ha! miecz obusieczny

**Pull request merged**